eng/pipelines/installer/jobs/base-job.yml

```diff
@@ -23,7 +23,7 @@ parameters:
   pgoType: ''
 
   packageDistroList:
-  - image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-14.04-debpkg-e5cf912-20175003025046
+  - image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-debpkg-20220408155542-cfdd435
```

Comment:
this appears to break the installer tests because of an ssl change?

Reply:
or I scanned to quickly and we need additional fixes for some

Reply:
I changed only one to see if the other also required changing. Looks like yes.